src/layouts/job.astro

```diff
@@ -0,0 +1,73 @@
+---
+import '@/styles/global.css';
+import { Navbar } from '@/components/ui/navbar';
+import Footer from '@/components/ui/footer.astro';
+
+const { title, description } = Astro.props;
+const canonicalUrl = new URL(Astro.url.pathname, Astro.site);
+---
+
+<!doctype html>
```

Comment:
A lof of this should be pulled from `<MainLayout title="Open Positions">` for consistency

Reply:
I think I fixed it?